gobblin-hive-registration/src/test/java/org/apache/gobblin/hive/metastore/HiveMetaStoreUtilsTest.java

```diff
@@ -165,4 +169,47 @@ public void testInVokeDetermineSchemaOrThrowExceptionMethod() {
       Assert.assertFalse(e instanceof NoSuchMethodException);
     }
   }
+
+  @Test
+  public void testgetHiveTable() throws Exception {
```

Comment:
hive-test.json is just a dummy file, since I need a sub-directory for testing hive snapshot. Git won't accept an empty directory. I changed it to a .gitignore file for maintaining a dummy directory since this way makes more sense.